src/PointwiseFunctions/AnalyticData/NewtonianEuler/SodExplosion.cpp

```diff
@@ -21,41 +21,34 @@ namespace NewtonianEuler::AnalyticData {
 template <size_t Dim>
 SodExplosion<Dim>::SodExplosion(const double initial_radius,
                                 const double inner_mass_density,
-                                const double inner_velocity,
                                 const double inner_pressure,
                                 const double outer_mass_density,
-                                const double outer_velocity,
                                 const double outer_pressure) noexcept
     : initial_radius_(initial_radius),
       inner_mass_density_(inner_mass_density),
-      inner_velocity_(inner_velocity),
       inner_pressure_(inner_pressure),
       outer_mass_density_(outer_mass_density),
-      outer_velocity_(outer_velocity),
       outer_pressure_(outer_pressure),
       equation_of_state_(1.4) {}
```

Comment:
Oops, missed this in my first pass: add ASSERTS that inner/outer densities/pressures are positive, for cases where constructed directly without option-parsing.